libs/designer/src/lib/ui/CustomNodes/OperationCardNode.tsx

```diff
@@ -74,6 +79,9 @@ const DefaultNode = ({ targetPosition = Position.Top, sourcePosition = Position.
   const isSecureInputsOutputs = useSecureInputsOutputs(id);
   const { status: statusRun, error: errorRun, code: codeRun, repetitionCount } = runData ?? {};
 
+  const suppressDefaultNodeSelect = useSuppressDefaultNodeSelectFunctionality();
+  const nodeSelectCallbackOverride = useAddedNodeSelectCallback();
```

Comment:
NIT: I don't know if `useAddedNodeSelectCallback` is the right name. I am just picky about the `Added` part

Reply:
Yeah I wasn't really sure how to phrase both of these, do you have any suggestions?

Reply:
Discussed offline, changing to `nodeSelectAdditionalCallback`